sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/UriDescribedType.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import java.net.URI;
+
+import static com.azure.messaging.servicebus.implementation.ServiceBusConstants.URI_SYMBOL;
+
+/**
+ * URI described type.
+ */
+public class UriDescribedType extends ServiceBusDescribedType {
+
+    /**
+     * Set described to describe data in described type.
+     *
+     * @param described  real value in the described type.
+     */
+    public UriDescribedType(Object described) {
```

Comment:
Could it take `described` as `URI` type?

Reply:
Yeah this class is used for convertint map provided by user may contain specific type to described type, the input value is  entry.getValue() and checked by using `instanceOf`, so it could take described as URI type.